frontend/components/AddPlayer/AddPlayer.jsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+import { useState } from 'react';
+import {
+  addToRoster,
+  createDummyRecord,
+  findUserByPhoneNumber,
+} from '@airtable/utils';
+import addStyle from './AddPlayer.module.css';
+import { formatPhoneNumber, createPlayerIdArray } from 'utils';
+
+const AddPlayer = ({ team, setTeamData }) => {
+  const [openView, setOpenView] = useState(false);
+  const [validationError, setValidationError] = useState('');
+  const { teamId, teamData } = team;
+
+  const toggleAddPlayerView = () => {
+    setOpenView(prev => !prev);
+    setValidationError('');
+  };
+
+  const handleChange = e => {
+    const input = e.target.value;
+    if (!input) setValidationError('');
+  };
+
+  const formValidation = e => {
+    e.preventDefault();
+    setValidationError('');
+    const phoneNumber = e.target.phone.value;
+    const pattern = /^\d+$/g;
```

Comment:
Can the var name explain what the pattern will be? For example, is it only numbers?